plugin-repos/starlark-apps/pixlet_renderer.py

```
@@ -0,0 +1,320 @@
+                for key, value in config.items():
+                    # Convert value to string for CLI
+                    if isinstance(value, bool):
+                        value_str = "true" if value else "false"
+                    else:
+                        value_str = str(value)
+                    cmd.extend(["-c", f"{key}={value_str}"])
+
+            logger.debug(f"Executing Pixlet: {' '.join(cmd)}")
+
+            # Execute rendering
+            result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)